src/store/transactions/index.ts

```diff
@@ -36,17 +36,23 @@ export const signedStacksTransactionState = atom(get => {
 });
 
 export const signedTransactionState = atom(get => {
-  const signedTransaction = get(signedStacksTransactionState);
-  if (!signedTransaction) return;
-  const serialized = signedTransaction.serialize();
-  const txRaw = stacksTransactionToHex(signedTransaction);
-  return {
-    serialized,
-    isSponsored: signedTransaction?.auth?.authType === AuthType.Sponsored,
-    nonce: signedTransaction?.auth.spendingCondition?.nonce.toNumber(),
-    fee: signedTransaction?.auth.spendingCondition?.fee?.toNumber(),
-    txRaw,
-  };
+  try {
+    const signedTransaction = get(signedStacksTransactionState);
+    if (!signedTransaction) return;
+    const serialized = signedTransaction.serialize();
```

Comment:
I suspect only this serialize and stacksTransactionToHex which calls serialize might raise an exception. Would it be better to try catch only those statements ?

Reply:
if anything in this block throws, we want to return undefined